src/lib/crowdinverify.ts

```diff
@@ -372,6 +372,8 @@ export async function crowdinVerify(member: GuildMember, url?: string | null, se
 }
 
 async function updateProjectRoles(projectName: ValidProjects, member: GuildMember, project: CrowdinProject) {
+	if (project.user_role === "pending") return
```

Comment:
this will always return false due to line 257

Reply:
can you explain why?